frontend/packages/integration-tests-cypress/cy-command-additions/selectors.ts

```diff
@@ -0,0 +1,20 @@
+import { timeouts } from '../utils/timeouts';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      getElemByDataID(selector: string, timeout?: number): Chainable<Element>;
+      getElemByActionID(selector: string): Chainable<Element>;
+    }
+  }
+}
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add('getElemByDataID', (selector: string, timeout: number = timeouts.default) =>
+  cy.get(`[data-test-id="${selector}"]`, { timeout }),
+);
+Cypress.Commands.add('getElemByActionID', (selector: string) =>
```

Comment:
I'd like to establish a crisp naming convention for these command since they'll be all over our tests. I'd suggest names like `Cypress.Commands.add('testID', ...)` and `Cypress.Commands.add('actionID', ...)` for common actions.

```ts
cy.testID('my-button').click();
```

seems better to me than


```ts
cy.getElemByDataID('my-button').click();
```

Reply:
Hmm...just switched it to `cy.getByTestID(...)` before I saw your comment.  Let me know if you still prefer `cy.testID(...)` -thanks

Reply:
`getTestID` is better than `getByTestID` imo

Reply:
fixed, renamed to `byTestID` and `byLegacyTestID`